src/components/forms/inputs/SelectInput.scss

```diff
@@ -1,10 +1,9 @@
-select{
-    @include input-base;
+select {
```

Comment:
Is it really a good idea to globally style `select` (and not `.SelectInput`)?

Reply:
No it isn't. All I did in this commit was to add a space, but we should make improvements when we notice them. Well spotted!

Reply:
🦅 👁